var/spack/repos/builtin/packages/libint/package.py

```diff
@@ -14,6 +15,7 @@ class Libint(AutotoolsPackage):
     homepage = "https://github.com/evaleev/libint"
     url = "https://github.com/evaleev/libint/archive/v2.1.0.tar.gz"
 
+    version('2.7.0-beta.1', sha256='149713e50e1f2f2669c3c5c2b1a47cf993f2cf3a10a02989cff487c9c80f316f')
```

Comment:
I wouldn't make a beta release be the default version. Add 2.6.0 and mark it as `preferred=True`.